src/styles/theme.js

```diff
@@ -149,13 +166,74 @@ export const themeChainsfr = createMuiTheme({
         }
       }
     },
+    MuiDialogActions: {
+      root: {
+        paddingTop: componentPaddings.modal.clearfix.none,
+        paddingBottom: componentPaddings.modal.default.paddingBottom,
+        paddingLeft: componentPaddings.modal.default.paddingLeft,
+        paddingRight: componentPaddings.modal.default.paddingRight
+      }
+    },
+    MuiDialogContent:{
+      root: {
+        paddingLeft: componentPaddings.modal.default.paddingLeft,
+        paddingRight: componentPaddings.modal.default.paddingRight,
+
+        '&.dialog-form': {
+          paddingTop: componentPaddings.modal.contentForm.paddingTop,
+          paddingBottom: componentPaddings.modal.contentForm.paddingBottom,
+        },
+
+        '&.dialog-content': {
+          paddingTop: componentPaddings.modal.clearfix.none,
+          paddingBottom: componentPaddings.modal.contentText.paddingBottom,
+        }
+      }
+    },
+    MuiDialogTitle:{
+      root: {
+        paddingTop: componentPaddings.modal.title.paddingTop,
+        paddingBottom: componentPaddings.modal.title.paddingBottom,
+        paddingLeft: componentPaddings.modal.default.paddingLeft,
+        paddingRight: componentPaddings.modal.title.paddingRight
+      }
+    },
+    MuiFormLabel: {
+      root: {
+        fontSize: 14
+      }
+    },
+    MuiIcon: {
+      root: {
+        fontSize: 12
+      },
+      fontSizeLarge: {
+        fontSize: 32
+      }
+    },
+    MuiInputBase: {
+      root: {   
+        fontSize: textValues.default.fontSize
+      }
+  },
     MuiMenuItem: {
       root: {
         fontSize: textValues.default.fontSize,
         lineHeight: textValues.default.lineHeight,
         color: textValues.default.color
       }
     },
+    MuiSvgIcon: {
+      fontSizeLarge: {
+          fontSize: 32
+      }
+    },
+    MuiTextField: {
```

Comment:
This would case TextField in transferForm to have extra padding

Reply:
fixed